lib/premailer/rails/customized_premailer.rb

```diff
@@ -12,7 +12,7 @@ def initialize(html)
         Premailer.send(:include, Adapter.find(Adapter.use))
         doc = load_html(html)
 
-        options = @options.merge(css_string: CSSHelper.css_for_doc(doc))
+        options = @options.merge(include_link_tags: false, css_string: CSSHelper.css_for_doc(doc))
```

Comment:
Since we are manually loading and embedding the css, we need to make sure premailer skips the link tags, otherwise it will load the assets through the network redundantly.